Source/WarpX.H

```diff
@@ -265,6 +274,12 @@ public:
 
     static amrex::Real quantum_xi_c2;
 
+
+    // Load balance options
+    enum LBType { UNDEFINED = -1, DEFAULT, HEURISTIC };
+    void LoadBalance (LBType load_balance_type = DEFAULT);
```

Comment:
Please add a Doxygen string to new APIs.
[Here is a small example](https://github.com/openPMD/openPMD-api/blob/a545854367624984555a1d2a06463e88796df3fd/include/openPMD/auxiliary/Date.hpp#L30-L35).

Reply:
Documentation added.